_63C/accesstokens.md

```diff
@@ -0,0 +1,8 @@
+---
+question: C8
+title: Are APIs and access tokens covered by FAL?
+---
+
+No, API access and any tokens used for API access are not covered by FAL. The FAL measurement is meant to cover the conveyance of identity information from an IdP to an RP through an assertion. This constitutes a federated login event to the RP. However, if that RP accesses an API, this is outside of the federated login. When that API dereferences or interprets the access token, this is not a login event since the user may or may not be present when the client is acting on the user's behalf. Therefore, both of these aspects are outside of FAL, and ultimately outside of 800-63.
```

Comment:
```suggestion
No, API access and any tokens used for API access are not covered by FAL. The FAL measurement is meant to cover the conveyance of identity information from an IdP to an RP through an assertion. This constitutes a federated login event to the RP. However, if that RP accesses an API, this is outside of the federated login. When that API dereferences or interprets the access token, this is not a login event since the user may or may not be present when the client is acting on the user's behalf. Therefore, both of these aspects are outside the scope of SP 800 C and the FAL designations.
```